$OpenBSD: patch-CMakeLists_txt,v 1.1.1.1 2020/03/03 21:15:51 sthen Exp $

- Use just -std=gnu99, not -std=c99 -std=gnu99.
- Some cmake fixes for finding libararies.

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -19,10 +19,6 @@ project(libavif LANGUAGES C VERSION 0.5.6)
 set(LIBRARY_VERSION "0.1.6")
 set(LIBRARY_SOVERSION "0")
 
-if(CMAKE_C_COMPILER_ID MATCHES "GNU")
-  add_definitions(-std=c99) # Enforce C99 for gcc
-endif()
-
 option(BUILD_SHARED_LIBS "Build shared avif library" ON)
 
 option(AVIF_CODEC_AOM "Use the AOM codec for encoding (and decoding if no other decoder is present)" OFF)
@@ -129,7 +125,7 @@ if(AVIF_CODEC_DAV1D)
         # Check to see if dav1d is independently being built by the outer CMake project
         if(NOT TARGET dav1d)
             find_package(dav1d REQUIRED)
-            include_directories(${DAV1D_INCLUDEDIR})
+            include_directories(${DAV1D_INCLUDE_DIR})
         endif()
         set(AVIF_CODEC_LIBRARIES ${AVIF_CODEC_LIBRARIES} ${DAV1D_LIBRARY})
     endif()
